wordpress-mobile/WordPress-Android
# [JP Social/Pre-publishing] Create social item UI

Part of #18694 

Creates Compose social item UI to be used in the pre-publishing sheet, in the Home fragment.

It also adds individual icon alpha support in `TrainOfIcons` and move some files around to better group classes in the `prepublishing` package.

To test:
Just Compose UI.

- Open repo
- Checkout this branch
- Go to `PrepublishingHomeSocialItem` file
- **Verify** preview shows up correctly
- Run the preview on a device
- **Verify** preview runs correctly on device

Besides that, you can also verify everything still works in the pre-publishing sheet after the class package changes.

## Regression Notes
1. Potential unintended areas of impact
Pre-publishing features breakage.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
